### ISSUE TITLE
Local Variable Paths Do Not Resolve Correctly

### ISSUE THREAD

**Issue by joshpoll** [penrose/penrose#645]
**Describe the bug**
Accessing properties of a local variable in a style program do not appear to resolve correctly.

**To Reproduce**

Domain: `setTheory.dsl`

Substance:
```
Set A
```

Style:
```
canvas {
  width = 800
  height = 700
}

Set x {
   arrow = Arrow { }

   encourage repelPt(10., arrow.start, arrow.end)
}
```

**Expected behavior**
The code should run without (compile time) errors.

**Actual behavior**
The following error message is produced:
`Error looking up path 'arrow.start' in translation. Name 'arrow' does not exist., Error looking up path 'arrow.end' in translation. Name 'arrow' does not exist.`

**Comment by samestep** [comment]
@joshpoll I'm guessing this is resolved as of #984?

**Comment by samestep** [comment]
I tried this on https://penrose.cs.cmu.edu/try/ just now (at commit 3f0947795e975c33f7d8cfad0be467746221f005) and can verify that it's fixed.

Domain:
```
type Set
```

Substance:
```
Set A
```

Style:
```
canvas {
  width = 800
  height = 700
}

forall Set x {
   arrow = Line { }
   encourage repelPt(10., arrow.start, arrow.end)
}
```
